lib/tasks/email_checkout_reminder.rake

```diff
@@ -0,0 +1,19 @@
+desc 'Send email reminder about upcoming checkout'
+task email_checkout_reminder: :environment do
+  if AppConfig.first.upcoming_checkout_email_active?
+    # get all reservations that start today
+    upcoming_reservations =
+      Reservation.where('checked_out IS NULL and '\
```

Comment:
let's use our scopes; this could be `Reservation.untouched.where('start_date = ?', Time.zone.today) instead. We could also technically use the `starts_on_days(Time.zone.today, Time.zone.today)` but I think the former is better.